components/GlobalSearch.tsx

```
@@ -13,8 +29,18 @@ export const GlobalSearch: React.FC<GlobalSearchProps> = ({
     <div className="flex gap-2 flex-wrap">
       <div className="relative flex-grow min-w-[200px]">
         <div className="absolute inset-y-0 left-0 pl-3 flex items-center pointer-events-none">
-          <svg className="h-4 w-4 text-secondary-text" fill="none" stroke="currentColor" viewBox="0 0 24 24">
-            <path strokeLinecap="round" strokeLinejoin="round" strokeWidth="2" d="M21 21l-6-6m2-5a7 7 0 11-14 0 7 7 0 0114 0z" />
+          <svg
+            className="h-4 w-4 text-secondary-text"
+            fill="none"
+            stroke="currentColor"
+            viewBox="0 0 24 24"
+          >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)